src/gui/optionsdlg.ui

```diff
@@ -3108,6 +3108,31 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
                  </layout>
                 </widget>
                </item>
+               <item>
+                <widget class="QGroupBox" name="groupAltWebUI">
```

Comment:
I would move this group above dynamic DNS group (under Authentication group).

Reply:
Ok.